frontend-react/src/App.tsx

```diff
@@ -98,6 +99,10 @@ const App = () => {
                                     path="/login/callback"
                                     component={LoginCallback}
                                 />
+                                <Route
```

Comment:
OK. So this form is NOT protected by an OKTA login and is available to the public, so we **will** get bad data sent to us.